libraries/Microsoft.Bot.Builder.AI.QnA/QnATelemetryConstants.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Default QnA event and property names logged using IBotTelemetryClient.
+    /// </summary>
+    public static class QnATelemetryConstants
+    {
+        public const string QnaMsgEvent = "QnaMessage"; // Event name
+        public const string KnowledgeBaseIdProperty = "knowledgeBaseId";
+        public const string AnswerProperty = "answer";
+        public const string ArticleFoundProperty = "articleFound";
+        public const string ChannelIdProperty = "channelId";
+        public const string OriginalQuestionProperty = "originalQuestion";
```

Comment:
We should make property names consistent between different recognizers. For the question properties, it is currently:
```C#
// LUIS
// Map to Activity.Text
public const string QuestionProperty = "questionProperty";

// QnA
// Map to Activity.Text
public const string OriginalQuestionProperty = "originalQuestion";
// Map to array of matched QnA questions
public const string QuestionProperty = "question";
```

It would be more consistent to use
```C#
// LUIS
// Map to Activity.Text
public const string QuestionProperty = "question";

// QnA
// Map to Activity.Text
public const string QuestionProperty = "question";
// Map to array of QnA questions matched
public const string MatchedQuestionProperty = "matchedQuestion";
```

Reply:
@ryanlengel : I spoke with Sreedhar.  He had a couple suggestions:
- Add QuestionId to QnA
- Remove MatchedQuestion 
- Add top 2 intents (and scores) to LuisResult

Any problems with this?  For reports, i'm guessing we want to keep matchedQuestion..

Reply:
We will definitely need matchedQuestion for the reports. QuestionId is already here, so that's good to go.

Top LUIS intents is a good idea, I guess in my usage I've only used queries that will always match one intent. I didn't realize it would return multiple but that makes sense!